elastic/kibana
# [SecuritySolution] Load entity store indices from security solution data view

## Summary

* Update the Entity Store to retrieve indices from the security solution data view.
* Create a new API that updates all installed entity engine indices (`api/entity_store/engines/apply_dataview_indices`)


### How to test it?
* Install the entity store
* Check if the transform index has the security solutions data view  indices
* Call `apply_dataview_indices` API; it should not return changes
* Update the security solution data view indices
* Call `apply_dataview_indices` API and if the API response contains the updated indices
* Check if the transform index also got updated
